config/testgrids/kubernetes/sig-cluster-lifecycle/config.yaml

```diff
@@ -49,9 +49,6 @@ dashboards:
     - name: capz-periodic-coverage-main
       test_group_name: periodic-cluster-api-provider-azure-coverage
       base_options: "exclude-filter-by-regex=Overall$&group-by-directory=&expand-groups=&sort-by-name="
-    - name: capz-pr-coverage-main
```

Comment:
Getting rid of the underlying `pull-cluster-api-provider-azure-coverage` job was giving complaints about this dashboard, and I was struggling to fix it. I then realized that the dashboard it refers to has been broken for a long time. See:

https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api-provider-azure#capz-pr-coverage-main

So, I think we can safely get rid of this for two reasons:

1. It doesn't really work
2. I'm not sure we want to track "PR coverage" over time, as various PRs may introduce various bits of signal that are not necessarily meaningful, and we want our PR gates to protect against coverage reductions anyways, not humans looking at a dashboard.